backend/lib/fetch/providers.py

```diff
@@ -1,16 +1,15 @@
 """Fetch data from database."""
+from sqlalchemy.orm import sessionmaker
+
+from backend.config import config
 from backend.lib import geocoder
 from backend.lib.database.postgres import connect, methods, postgis
 from backend.lib.database.tables import address
 from backend.lib.timer import timed
-from backend.models import distance
-
-from sqlalchemy.orm import sessionmaker
 
-# TODO: Use config or environment variable.
-GEOCODING = True
-DEFAULT_GEOCODER = 'oxcoder'
-MEASURER = distance.get_measure('haversine')
+# Use values from configuration if they exist.
+GEOCODING = config.get('geocoding_enabled')
```

Comment:
Open question, should we keep this or use the config calls directly further down in the file?